stable/mattermost-team-edition/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Mattermost Team Edition server.
 name: mattermost-team-edition
-version: 2.2.1
+version: 2.3.0
```

since is just a small fix, just the minor
```suggestion
version: 2.2.2
```

---

it's small fix but changes to user experience (push wasn't working because it was trying to use enterprise push server)

---

yep, but sounds more a bug fix

---

@ahmadalli thanks. I will fix the DCO